common/constant.js

```diff
@@ -0,0 +1,40 @@
+const OTP = Math.floor(100000 + Math.random() * 900000);
+const SALT_ROUNDS = 10;
+const SUCCESSFULLY_LOGIN = 'You have successfully login.';
+const USER_ID_NOT_FOUND = { message: "User id not found." };
```

Comment:
why you use  `{ message: "User id not found." }`? 

it should be like this "User id not found"